### PR TITLE
fix: apply foundation audit

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch would:
- fix omitted invariant
- fix unit tests and refactor them.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: proto/lbm/foundation/v1/foundation.proto [hunk 1]
```diff
@@ -27,7 +27,7 @@ message Member {
   string metadata = 2;
 
   // added_at is a timestamp specifying when a member was added.
-  google.protobuf.Timestamp added_at = 4 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  google.protobuf.Timestamp added_at = 3 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
```
Comment (zemyblue) [inline proto/lbm/foundation/v1/foundation.proto:30]: I think this is breaking change. 
Is this change required?

Comment (0Tech) [reply proto/lbm/foundation/v1/foundation.proto]: OK, I will revert it.

**Review by zemyblue — APPROVED** [review]

**Review by dudong2 — APPROVED** [review]
